src/main/java/xyz/aadev/generitech/common/blocks/Blocks.java

```diff
@@ -86,9 +92,11 @@ public ItemStack getStack() {
         return new ItemStack(block);
     }
 
+/*
     public ItemStack getStack(int size) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)